packages/core/src/components/forms/_control-group.scss

```diff
@@ -96,6 +96,12 @@ Styleguide control-group
     }
   }
 
+  // Add border radius inheritance to support components wrapped in a popover
+  .#{$ns}-popover-wrapper,
+  .#{$ns}-popover-target {
+    border-radius: inherit;
+  }
```

Comment:
please move this next to the border-radius styles around line 188. this location is smack in the middle of a whole bunch of z-index logic.